aries-site/src/pages/templates/matrix-navigation.mdx

```diff
@@ -1,3 +1,42 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
 
-<ComingSoon />
\ No newline at end of file
+## When a User would Use
+- Pages where the hierarchy isn't crucial to the user experience, and where browsing is encouraged. For ex: HPE.com
+- User flows where a global nav bar is visible at all times.
+
+### Matrix within a Music App Example
+An example of a music app where the playlist is the hub and a song is the spoke. From the song, a user 
+can matrix navigate to the artist or album. 
+
+<Example 
+  caption="Illustration showing how the user can matrix navigate to a song from an album within a music application." 
+  plain 
+  pad="small"
+  previewWidth="large"
+  height={{ min: 'small' }}
+>
+  <DiagramPreview 
+    src={{
+      dark: "/templateImages/matrix-music-invert.svg",
+      light: '/templateImages/matrix-music.svg'
+    }} 
+    alt="Diagram illustrating the matrix navigation in a music app." />
+</Example>
+
+### Matrix within App Example
+Users can search for information within a set of cards to select or navigate to different access points in no particualr order.
+
+<Example 
+  caption="Illustration showing how a user can matrix navigate within an application." 
```

Comment:
The caption should explain what the illustration/diagram is trying to explain. 

Another way to think about it is "if I didn't have the illustration, how would I describe the concept in words?" Or, "what is the key message/point a reader should takeaway from the diagram?"

To the latter, I'm not sure what this diagram is illustrating or what the key point is... Is it trying to show me that dashboard, applications, profile, and help are always accessible navigation points to me?